src/components/Avatar/README.md

```diff
@@ -0,0 +1,4 @@
+## Avatar
```

Comment:
I recommend taking a look at other README's, and using the same format for this one.

Reply:
Yes, but this one is different from others, as we do not have any re-usable theme properties here, right?

Reply:
Nevertheless this should be edited, and not left with the default generated text.
You can write a short description, and how to use.
Also you can add a link that its implementation is used with the `wix-ui-core` component.

Reply:
Added usage example and links to wix-ui-core;